Python-Home-Challenges/app.py

```diff
@@ -0,0 +1,10 @@
+f = open("Challenge_01.txt", "r")
+data = f.read().split()
+dict = {}
```

Comment:
variable names should be clear and meaningful.

Reply:
f is now file
data is now fileArr
dict is now words